src/commonTest/kotlin/fr/acinq/lightning/serialization/StateSerializationTestsCommon.kt

```diff
@@ -38,14 +40,23 @@ class StateSerializationTestsCommon : LightningTestSuite() {
+        )
+        val state = Serialization.deserialize(bin, TestConstants.Alice.nodeParams)
+        println(state)
```

Comment:
Replace with a real test?

Reply:
there is one in `TestHelpers.processEx()`, I keep this one because it was generated with version `v1.0-beta11` but it's obsoleted by https://github.com/ACINQ/lightning-kmp/pull/259